app/forms.py

```diff
@@ -97,6 +98,56 @@ def save(self, *args, **kwargs):
         send_sms.delay(self.profile.mobile_number, message)
 
 
+class PasswordResetEmailForm(forms.Form):
+    email = forms.CharField(max_length=64)
+
+        
+    def clean_email(self):
+        """
+        Validates that an active user exists with the given email address.
+        """
+        email = self.cleaned_data['email']
+        # Fail with invalid number.
+        try:
+            email = self.cleaned_data["email"]
+            self.user = User.objects.get(email__iexact=email)
+        except app.models.UserProfile.DoesNotExist:
+            raise forms.ValidationError("Unable to find an account for the "
+                                        "provided email. Please try "
+                                        "again.")
+        return email
+    def save(self, *args, **kwargs):
+        """
+        Generates a one-use only link for resetting password and sends to the
+        user.
+        """
+
+        uid = int_to_base36(self.user.id)
+        token = default_token_generator.make_token(self.user)
+        current_site = get_current_site(kwargs['request'])
+
+        message = "Hi %s. Follow this link to reset your pin: http://%s%s" % (
+            self.user.username,
+            current_site.domain,
+            reverse(
+                'password_reset_confirm',
+                kwargs={'uidb36': uid, 'token': token}
+            )
+        )
+
+        subject = "Password Reset"
+        recipients= [self.user.email]
+        from_address = settings.FROM_EMAIL_ADDRESS
```

Comment:
Please add a default in `settings.py` so that this doesn't fail. We'll overwrite it on deploy.